unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/Controllers/BIWCreatorController.cs

```diff
@@ -193,6 +193,15 @@ public void RemoveLoadingObject(string entityId)
         loadingPlaceHolder.DestroyAfterAnimation();
     }
 
+    public void RemoveLoadingObjectInmediate(string entityId)
+    {
+        if (!loadingGameObjects.ContainsKey(entityId))
+            return;
```

Comment:
You could use the `TryGetValue` method here:

Something like
```suggestion
        if (!loadingGameObjects.TryGetValue(entityId, out BIWLoadingPlaceHolder loadingPlaceHolder))
            return;
```

It's quite handy :). In any case this is fine, you dont have to change it